deployments/lookup_hints.yaml

```
@@ -2,31 +2,38 @@ status: Converted
 registry_authors:
 - Nicolas Berrios
 - Jack Fitzsimons
+- James Honaker
 tier: 1
 deployment:
   name: Lookup Hints
   data_curator: Apple
   intended_use: Web Browsing
   data_product_type: Summary statistics # TODO: Is this correct?
   data_product_region: Not specified
-  description: For Lookup Hints, Apple uses a privacy budget with epsilon
-    of 4, and limits user contributions to two donations per day.
-  publication_date: '1999-01-01' # TODO: Is this correct?
+  description: 'The Apple differential privacy implementation incorporates the concept of a per-
+    donation privacy budget (quantified by the parameter epsilon), and sets a strict limit on
+    the number of contributions from a user in order to preserve their privacy... For Lookup Hints, Apple uses a privacy budget with epsilon
+    of 4, and limits user contributions to two donations per day.'
```

```suggestion
  description: |
    The Apple differential privacy implementation incorporates the concept of a per-
    donation privacy budget (quantified by the parameter epsilon), and sets a strict limit on
    the number of contributions from a user in order to preserve their privacy... For Lookup Hints, Apple uses a privacy budget with epsilon
    of 4, and limits user contributions to two donations per day.
```

Slight preference to use block quotes for multilines, so then either `'` or `"` can be used without a problem.